src/public/modules/forms/helpers/CsvMergedHeadersGenerator.ts

```diff
@@ -80,9 +98,14 @@ class CsvMergedHeadersGenerator extends CsvGenerator {
    * Extracts the string representation from an unprocessed record.
    * @param {Object} unprocessedRecord
    * @param {string} fieldId
+   * @param {number} colIndex
    * @returns {string}
    */
-  _extractAnswer(unprocessedRecord, fieldId, colIndex) {
+  _extractAnswer(
+    unprocessedRecord: Dictionary<Response>,
```

Comment:
the problem with `Dictionary` is that it assumes that every string is a valid key. for example, if you do `const value = unprocessedRecord['some crap']`, `value` will be typed as a `Response`. so instead of using `Dictionary` it would be better to declare our own specific type.